tests/e2e/tests/simple/servicesToBeInjected.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: echosrv
+spec:
+  ports:
+  - port: 8080
+    name: http-echo
+  - port: 8079
+    name: grpc-ping
+  selector:
+    app: echosrv
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: simple-ingress
+  annotations:
+    kubernetes.io/ingress.class: istio
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /.*
```

Comment:
doesn't seem to work (as I expect) with a different path e.g /fortio/.* if I call fortio/debug it doesn't work (as in it doesn't call fortio with /debug)